lib/tab-bar-item-ios.js

```diff
@@ -16,18 +16,22 @@ export default function createTabBarItemIOSComponent(IconNamePropType, getImageS
       iconName: IconNamePropType.isRequired,
       selectedIconName: IconNamePropType,
       iconSize: PropTypes.number,
+      iconColor: PropTypes.string,
+      selectedIconColor: PropTypes.string,
     };
 
     static defaultProps = {
       iconSize: 30,
+      iconColor: '#929292',
+      selectedIconColor: '#929292',
     };
 
     updateIconSources(props) {
       if (props.iconName) {
-        getImageSource(props.iconName, props.iconSize).then(icon => this.setState({ icon }));
+        getImageSource(props.iconName, props.iconSize, props.iconColor).then(icon => this.setState({ icon }));
       }
       if (props.selectedIconName) {
```

Comment:
It should also cover the case where 
* `selectedIconName` is `undefined` but `selectedIconColor` is not. 
* `selectedIconColor` is `undefined` but `selectedIconName` and `iconColor ` are not.